jobs/kpi-forecasting/Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN groupadd --gid ${USER_ID} ${GROUP_ID} && \
 
 WORKDIR ${HOME}
 
+RUN apt-get -y update
```

Comment:
So the best practice would to combine commands into a single layer if possible.

In this case, we could do the following:
```
RUN apt-get -y update && apt-get -y install \
    git \
    && rm -rf /var/lib/apt/lists/*
```

Here's a Stackoverflow comment about this:
https://stackoverflow.com/a/39330458

Reply:
Thank you, it makes sense now that you point it out but I haven't had enough experience with building Docker containers to catch that. I've made this change.